onchain/wallet/wallet.go

```diff
@@ -229,10 +233,15 @@ func Initialized() bool {
 }
 
 func Init(walletConfig Config) error {
-
 	if config != nil {
 		return errors.New("already initialized")
 	}
+	if walletConfig.AutoConsolidateThreshold == 0 {
+		walletConfig.AutoConsolidateThreshold = DefaultAutoConsolidateThreshold
+	}
```

Comment:
Let's apply the default in the config and don't auto consolidate when the configured number is 0

Reply:
Not sure if we should allow to opt out of that since it makes the wallet unusable once the max has been reached